pkg/auth/auth.go

```diff
@@ -84,9 +85,9 @@ func Login(ctx context.Context, systemContext *types.SystemContext, opts *LoginO
 		}
 		logrus.Debugf("registry not specified, default to the first registry %q from registries.conf", server)
 	} else {
-		server = getRegistryName(args[0])
+		server = trimScheme(args[0])
```

Comment:
`server` will no longer be a good name for this variable.

Reply:
Fixed